composer.json

```diff
@@ -6,10 +6,12 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
+        "php": "^7.1",
         "drupal/administration_language_negotiation": "1.4",
         "drupal/core": "~8.6",
-        "drupal/pathauto": "1.2",
-        "php": "^7.1"
+        "drupal/devel": "^1.2",
+        "drupal/language_selection_page": "2.2",
+        "drupal/pathauto": "1.2"
```

Comment:
@pfrenssen agreed, this will also allow site developers to fix the version they want to use in the site's `composer.json`, in case they need it, while if we lock them on the components' `composer.json` that would not be possible.

Reply:
Regarding the `~2.2` and `^2.2` both are valid so they are both OK to be used. I don't think there is a need to define a policy because there is no "better" way to do this. Both cases are really easy to understand: the tilde updates only the most minor version specified, while the caret updates all versions except the most major one.

I personally prefer the tilde for dependencies that use a `<major version>.<minor version>` naming scheme such as Drupal modules. The caret is definitely the one to use for packages that adhere to semantic versioning `<major version>.<minor version>.<patch>`.

Some people like to use the caret for everything because it is "less confusing for newbies", but I think it is condescending to insinuate that young developers are not able to understand the difference between the two.

Reply:
This has been addressed, thanks :+1:

I see that the `drupal/administration_language_negotiation` dependency is also locked but this is out of scope for this ticket, I created a followup to do this: OPENEUROPA-601.